en/admin/db_manage.rst

```diff
@@ -57,9 +57,9 @@ The volumes of CUBRID database are classified as permanent volume, temporary vol
 *   In the permanent volumes,
 
     *   there are data volumes, that usually store permanent data, but can also store temporary data.
-    *   there are log volumes, that can be further classified as: one active log, archive logs and one background archiving log.
+    *   there is the log volume, that can be further classified as: one active log.
     
-*   In the temporary volumes, only temporary data is stored.
+*   In the temporary volumes, temporary data, archive logs and one background archiving log are stored.
```

Comment:
How about referring to the permanent section? It would be nice to explain temporary data and archive logs separately. And the background archiving log has no temporary characteristics as I know. I don't think there's any need to mention the background archive log. However, since it was mentioned before the revision, it cannot be removed arbitrarily. I will check the code anyway.